contracts/clients/package.json

```diff
@@ -9,7 +9,7 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "build": "rm -rf dist && mkdir dist && cp -rH typechain dist/typechain && tsc",
+    "build": "rm -rf dist && mkdir dist && cp -rH typechain dist/typechain && find ./dist/typechain -type f \\! -name '*.d.ts' -name '*.ts' -delete && tsc",
```

Comment:
Avoids publishing non-definition typescript files from typechain? Nice. Could you add something about this to the PR description?

Reply:
Added to description.